lib/eval_game.rb

```diff
@@ -2,10 +2,6 @@ module TicTacToe
 
   class EvalGame
```

Comment:
We usually use nouns for class names and verbs for method names. Maybe change this name to something like GameEval or GameEvalution